h/templates/help.html.jinja2

```diff
@@ -7,12 +7,16 @@
   {{ super() }}
   <script type="application/json" class="js-hypothesis-config">
     {
-      "firstRun": true, {# TODO: remove after 2016-09-01, by which time old clients will be gone. #}
       "openSidebar": true,
       "openLoginForm": true
     }
   </script>
   {% if not is_onboarding %}
+    <script type="application/json" class="js-hypothesis-config">
```

Comment:
This is in a separate config block so that we only set the sidebar app URL if the client is being embedded on the page.